desc/protoparse/lexer.go

```diff
@@ -209,8 +209,8 @@ func (l *protoLex) Lex(lval *protoSymType) int {
 			// we're not actually returning a rune, but this will associate
 			// accumulated comments as a trailing comment on last symbol
 			// (if appropriate)
-			l.setIdent(lval, "")
-			l.eof = lval.id
+			l.setRune(lval, 0)
```

Comment:
The EOF needs to be considered a rune in order for the trailing comment attribution logic to work correctly.